xcodeproj/compile_only_aspect.bzl

```diff
@@ -65,18 +63,16 @@ def _compile_only_aspect_impl(target, ctx):
         )
     else:
         return []
-
     for dep in deps:
         if OutputGroupInfo in dep and not hasattr(dep[OutputGroupInfo], "compiles"):
-            fail(target, dep)
-
+            continue
```

Comment:
If this isn't to signal the failure anymore, it can just be removed.
```suggestion
```

Reply:
Ah yes, makes sense.